### PR TITLE
Remove dead code for favorite servers

### PR DESCRIPTION
**Pull request by slipher**
Companion to https://github.com/DaemonEngine/Daemon/pull/775

### CODE REVIEW

**Comment by DolceTriade** [comment]
We should remove the favorite code from the master server too if we want to do this.

**Comment by slipher** [comment]
> We should remove the favorite code from the master server too if we want to do this.

Maybe you are thinking of "featured" servers. Favorite servers were the user's personal favorites stored in a file in the homepath.

**Comment by DolceTriade** [comment]
Ah, both should be removed then :D this lgtm

File: src/cgame/cg_rocket_datasource.cpp [hunk 1]
```diff
@@ -427,7 +418,7 @@ void CG_Rocket_CleanUpServerList( const char *table )
 	int j;
 	int netSrc = CG_StringToNetSource( table );
 
-	for ( i = AS_LOCAL; i <= AS_FAVORITES; ++i )
+	for ( i = AS_LOCAL; i <= AS_NUM_TYPES; ++i )
```
Comment (necessarily-equal) [inline src/cgame/cg_rocket_datasource.cpp:421]: Is this on purpose or did you mean this instead?
```suggestion
	for ( i = AS_LOCAL; i < AS_NUM_TYPES; ++i )
```

File: src/cgame/cg_rocket_datasource.cpp [hunk 2]
```diff
@@ -244,83 +244,74 @@ void CG_Rocket_BuildServerList( const char *args )
...
+		if ( rocketInfo.data.retrievingServers )
+		{
+			rocketInfo.data.retrievingServers = false;
 		}
-
-	}
-
-	else if ( !Q_stricmp( args, "serverInfo" ) )
-	{
-		CG_Rocket_BuildServerInfo();
```
Comment (necessarily-equal) [inline src/cgame/cg_rocket_datasource.cpp:323]: Was this deletion intended?

**Review by necessarily-equal — COMMENTED** [review]
Not sure why the build fails. I've made comments for two oddities

File: src/cgame/cg_rocket_datasource.cpp (hunk 1 above)
Comment (slipher) [reply src/cgame/cg_rocket_datasource.cpp]: Thanks, fixed

**Pull request merged**